script/script.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable linebreak-style */
+const navWrapper = document.querySelector('.hamburger-menu');
+const hamMenu = document.querySelectorAll('.line');
+const sideNav = document.querySelector('.side-nav');
+const sideLinks = document.querySelectorAll('.links');
+const Toggler = (e) => {
+  e.preventDefault();
+  hamMenu.forEach((a) => a.classList.toggle('active'));
+  sideNav.classList.toggle('hide');
+};
+
+navWrapper.addEventListener('click', Toggler);
+
+// eslint-disable-next-line no-plusplus
+for (let i = 0; i < sideLinks.length; i++) {
+  sideLinks[i].addEventListener('click', (e) => {
+    const id = sideLinks[i].href.slice(23, sideLinks[i].href.length);
+    Toggler(e);
+    document.getElementById(id).scrollIntoView();
+  });
```

Comment:
- [ ] Clicking on links is closing the menu ✔️ However now they don't take us to the right section as it did for the previous reviewer so there is a new bug in the code, kindly fix this issue 👍 

 
![Animation](https://user-images.githubusercontent.com/50721479/201925952-32be50e7-fed3-43fc-83c1-13563d5a3508.gif)

Reply:
Kindly check again because it is already working on both Chrome and Mozilla. here is the link to loom, https://www.loom.com/share/3a1b4a2aaf1e408d8fb365955be3092d.
( You can see that I haven't committed any new code so it was already working)